### DISCUSSION TITLE
How did you integrate vscode into the app?

### DISCUSSION THREAD

**Discussion by jameszhang-a** [blueedgetechno/win11React#11] [Q&A] [answered]
First of, this is one of the coolest project's I've seen, especially considering it took you a week. 

How did you get the browser and vscode in your app? I'm fairly new to React and webdev, but after going through your code, I still have no idea how you did that haha.

**Comment by blueedgetechno** [comment, marked as answer]
There is this awesome open source project, [github1s](https://github.com/conwnet/github1s) which tries to simulate vscode in browser. I was getting too lazy to make another app from scratch. So I just ... ᶦᶠʳᵃᵐᵉᵈ ᶦᵗ. Lol

**Reply by jameszhang-a** [reply]
Haha still cool! No reason to rebuild something that complicated if it's already been done right?

**Reply by blueedgetechno** [reply]
I mean, isn't that's why open source exist.
